src/views/sandwich-view.tsx

```diff
@@ -30,19 +76,50 @@ class SandwichView extends StatelessComponent<SandwichViewProps> {
   }
 
   componentDidMount() {
+    window.addEventListener('mousemove', this.resizeViews)
+    window.addEventListener('mouseup', this.stopResize)
+    window.addEventListener('mouseleave', this.stopResize)
     window.addEventListener('keydown', this.onWindowKeyPress)
   }
   componentWillUnmount() {
     window.removeEventListener('keydown', this.onWindowKeyPress)
+    window.removeEventListener('mouseleave', this.stopResize)
+    window.removeEventListener('mouseup', this.stopResize)
+    window.removeEventListener('mousemove', this.resizeViews)
   }
 
   render() {
     const {selectedFrame} = this.props
+    let container: JSX.Element | null = null
+    let functionsView: JSX.Element | null = null
+    let resizer: JSX.Element | null = null
     let flamegraphViews: JSX.Element | null = null
 
     if (selectedFrame) {
+      functionsView = (
+        <div
+          className={css(style.tableView)}
+          style={{
+            width: `calc(100% - ${this.state.resizerPos ? this.state.resizerPos + 'px' : '50%'})`,
+          }}
+        >
+          <ProfileTableViewContainer activeProfileState={this.props.activeProfileState} />
+        </div>
+      )
+      resizer = (
+        <div
+          className={css(style.resizer)}
+          style={{left: this.state.delta}}
+          onMouseDown={e => this.startResize(e)}
+        />
+      )
       flamegraphViews = (
-        <div className={css(commonStyle.fillY, style.callersAndCallees, commonStyle.vbox)}>
+        <div
+          className={css(commonStyle.fillY, style.callersAndCallees, commonStyle.vbox)}
+          style={{
+            width: `calc(100% - ${this.state.resizerPos ? this.state.resizerPos + 'px' : '50%'})`,
```

Comment:
Resizing the view isn't sufficient to update the flamechart view, resulting in it not re-rendering until mouse over. This creates visual artifacts that are avoidable by (somehow) forcing the flamechart view to re-render.

Currently the `FlamechartPanZoomView` listens for window resize events to re-render:

https://github.com/jlfwong/speedscope/blob/30ca6291ca93557e2bb77a2bafc5bac580627102/src/views/flamechart-pan-zoom-view.tsx#L699

https://github.com/jlfwong/speedscope/blob/30ca6291ca93557e2bb77a2bafc5bac580627102/src/views/flamechart-pan-zoom-view.tsx#L404-L407

I haven't tried this, but one way to fix this might be to fake a window resize event, like this:

```
window.dispatchEvent(new UIEvent('resize'))
```

I didn't fix this in the code I wrote and attached to the top-level PR